python/dgl/graphbolt/impl/csc_sampling_graph.py

```diff
@@ -444,6 +444,9 @@ def sample_neighbors(
         return self._convert_to_sampled_subgraph(C_sampled_subgraph)
 
     def _check_sampler_arguments(self, nodes, fanouts, probs_name):
+        assert (
```

Comment:
If required, we can move this ahead to python.

Reply:
where do mean exactly? checking here looks ok to me as it's for sample argument

Reply:
removed such check as we don't need it any more in the latest implementation.